### ISSUE TITLE
Radio label click not working

### ISSUE THREAD

**Issue by nguyentranchung** [protonemedia/laravel-form-components#9]
![image](https://user-images.githubusercontent.com/9611224/92377145-a531eb80-f13e-11ea-8124-08c1a5a68a8c.png)
In your example, click radio label not working.

See bootstrap 4 radio example
```html
<div class="form-check form-check-inline">
  <input class="form-check-input" type="radio" name="inlineRadioOptions" id="inlineRadio1" value="option1">
  <label class="form-check-label" for="inlineRadio1">1</label>
</div>
<div class="form-check form-check-inline">
  <input class="form-check-input" type="radio" name="inlineRadioOptions" id="inlineRadio2" value="option2">
  <label class="form-check-label" for="inlineRadio2">2</label>
</div>
```

**Comment by pascalbaljet** [comment]
Thanks for noticing, I'll take a look at your PR soon.

**Comment by pascalbaljet** [comment]
Fixed in #13.

Since this is breaking change, you need to upgrade to `v2.0.0`.
